include/benchmark/benchmark.h

```diff
@@ -712,11 +712,9 @@ enum Skipped
 #if defined(BENCHMARK_HAS_CXX11)
     : unsigned
 #endif
-{
-  NotSkipped = 0,
+{ NotSkipped = 0,
```

Comment:
this doesn't look right.. did clang-format with Google style make this change?

Reply:
I think yes I just ran the clang-format on this file too and I remember before making the commit I didn't make any change in benchmark.h file beside moving the OneK enum from Counter class.

I even given a second thought before pushing these changes, I'll run the clang-format again.

Reply:
I again ran the command  `clang-format -i -style=Google /home/devansh/benchmark/include/benchmark/benchmark.h` but the clang-format made no changes in the file.

![image](https://github.com/google/benchmark/assets/20411128/f15d780c-c2ad-4594-807d-e4024bdad784)

Reply:
maybe you can just revert it manually then

Reply:
okay, doing it.